### PR TITLE
MAT-11019: allow cron job to be triggered manually

### PR DESCRIPTION
**Pull request by mcmcphillips**
## Terminology Service PR

Jira Ticket: [MAT-11019](https://jira.cms.gov/browse/MAT-11019)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/terminology/controller/AdminController.java [hunk 1]
```diff
@@ -91,4 +93,18 @@ public ResponseEntity<List<String>> evictAllCaches(Principal principal) {
     evictedCaches.forEach(cacheName -> cacheManager.getCache(cacheName).clear());
     return ResponseEntity.ok(evictedCaches);
   }
+
+  @PostMapping(path = "/trigger-code-system-refresh")
+  @PreAuthorize("hasRole('MADIE-ADMIN')")
+  public ResponseEntity<String> triggerCodeSystemRefresh() {
+    if (!updateCodeSystemTask.isRunning()) {
+
+      new Thread(() -> updateCodeSystemTask.updateCodeSystems()).start();
```
Comment (RohitKandimalla) [inline src/main/java/gov/cms/madie/terminology/controller/AdminController.java]: Is there a reason, we explicitly need to create a manual thread ? Looks like the UpdateCodeSystemTask already has the AtomicBoolean which handles concurrent requests. I am thinking the Controller doesn't need to create new threads manually

**Review by RohitKandimalla — APPROVED** [review]
